scripts/run_vm.sh

```diff
@@ -74,11 +74,11 @@ function start {
 
   if [ "${ELMNTL_DEBUG}" == "yes" ]; then
       qemu-system-${ELMNTL_TARGETARCH} ${kvm_arg} ${disk_arg} ${cdrom_arg} ${global_arg} ${firmware_arg} \
-          ${usrnet_arg} ${kvm_arg} ${memory_arg} ${graphics_arg} -serial stdio ${pidfile_arg} \
+          ${usrnet_arg} ${memory_arg} ${graphics_arg} -serial stdio ${pidfile_arg} \
```

Comment:
`kvm_arg` arguments were included twice. See this is already present in the line above.